src/ast/passes/codegen_llvm.cpp

```diff
@@ -1297,7 +1297,7 @@ void CodegenLLVM::visit(Call &call)
     if (!map_len_func_)
       map_len_func_ = createMapLenCallback();
 
-    b_.CreateForEachMapElem(ctx_, map, map_len_func_, len, call.loc);
+    b_.CreateForEachMapElem(ctx_, map, map_len_func_, len, len, call.loc);
```

Comment:
I think the `len` allocated on stack is no longer necessary. Can you try using a null ptr instead?

Reply:
Yes, you are right.  I'll revise it.